ListsProject.Tests/ArrayListTests.cs

```diff
@@ -0,0 +1,418 @@
+using NUnit.Framework;
+using System;
+
+namespace ListsProject.Tests
+{
+    public class ArrayListTests
```

Comment:
Please make one test class for all collections https://github.com/UMSF-K13-1-Samoylenko/LibraryGitTutorial/tree/ForReview